widgetastic_manageiq/__init__.py

```diff
@@ -38,7 +38,7 @@
 from widgetastic.xpath import quote
 from widgetastic_patternfly import (
     Accordion as PFAccordion, CandidateNotFound, BootstrapSwitch, BootstrapTreeview, Button, Input,
-    BootstrapSelect, CheckableBootstrapTreeview, FlashMessages)
+    BootstrapSelect, CheckableBootstrapTreeview, FlashMessages, Dropdown, VerticalNavigation)
```

Comment:
Prefer sorting imports alphabetically for readability.

Reply:
Done